.gitignore

```diff
@@ -41,6 +41,7 @@ _UpgradeReport_Files/
 ipch/
 *.sdf
 *.opensdf
+node.VC.opendb
```

Comment:
And FWIW, this seems to be a common thing to add to .gitignore files.  e.g.,http://stackoverflow.com/questions/34975423/visual-studio-2015-git-error-opensomefile-vc-opendb-permission-denied-fa